### ISSUE TITLE
Camara data base

### ISSUE THREAD

**Issue by pickettph** [alicevision/meshroom#280]
I have a sony cyber-shot dsc-h1 and the data I can find on it says that its sensor is6.104 by 4.578mm.  How can this be added to the database?

**Comment by Powerbyte7** [comment]
Your camera sensor is already in the camersensors.db file. So I'd check #179 to see what stops it from working.

This is how you would normally go about it:
As discussed in #226 you need to add a line to the camersensors.db file.
According to digicamdb.com the sensor size is 5.75 mm
Meaning the line you need to add is:

> Sony;Sony Cybershot DSC H1;5.75

**Comment by pickettph** [comment]
Thank you, how ever it is still not working.  Is it possible that the problem is it image size?  I see no way to change it.  I'm stuck.

![meshroom1](https://user-images.githubusercontent.com/44305456/47257479-57ce0780-d45c-11e8-8c7b-60dc754f9dd6.png)
![meshroom2](https://user-images.githubusercontent.com/44305456/47257480-5a306180-d45c-11e8-8491-14e69c7ecc37.png)

**Comment by natowi** [comment]
Go to the sensor database in Meshroom-2018.1.0\aliceVision\share\aliceVision and edit cameraSensors.db
There is Sony;Sony Cybershot DSC H1;5.75 in the database, but Meshroom searches the model in EXIF data. This can differ. So try to add the following line to the database: Sony;DSC-H1;5.75
save and restart Meshroom

**Comment by pickettph** [comment]
Thank you for your help and sorry it took so long to get back to you.  I found mk breuer's 50 photo zip file and tried it, no go.  Reinstalled  and get his rock working. Tried the sony camara, no go so added Sony;DSC-H1;5.75 to the database and saved as an odf file, still no go. Tried changing the data base this time saving as a text file, it works.  Do I need to do a pull, or is that something you do to close this matter?
